biojava-structure/src/main/java/org/biojava/nbio/structure/geometry/SuperPositionQCP.java

```diff
@@ -0,0 +1,441 @@
+/*
+ *                    BioJava development code
+ *
+ * This code may be freely distributed and modified under the
+ * terms of the GNU Lesser General Public Licence.  This should
+ * be distributed with the code.  If you do not have a copy,
+ * see:
+ *
+ *      http://www.gnu.org/copyleft/lesser.html
+ *
+ * Copyright for this code is held jointly by the individual
+ * authors.  These should be listed in @author doc comments.
+ *
+ * For more information on the BioJava project and its aims,
+ * or to join the biojava-l mailing list, visit the home page
+ * at:
+ *
+ *      http://www.biojava.org/
+ *
+ */
+
+package org.biojava.nbio.structure.geometry;
+
+import javax.vecmath.Matrix3d;
+import javax.vecmath.Matrix4d;
+import javax.vecmath.Point3d;
+import javax.vecmath.Vector3d;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Implementation of the Quaternion-Based Characteristic Polynomial algorithm
+ * for RMSD and Superposition calculations.
+ * <p>
+ * Reference: Theobald DL (2005). Rapid calculation of RMSDs using a
+ * quaternion-based characteristic polynomial. Acta Crystallogr A. 2005
+ * Jul;61(Pt 4):478-80. (PubMed id: 15973002)
+ * 
+ * @author Peter Rose
+ * @author Aleix Lafita
+ * 
+ */
+public final class SuperPositionQCP {
+	
+	private static final Logger logger = LoggerFactory
+			.getLogger(SuperPositionQCP.class);
+
+	double evecprec = 1E-6;
+	double evalprec = 1E-11;
```

Comment:
It would be very useful if these 2 were parameters could be set externally, having the current values as the default if they aren't set explicitly. For many applications a very rough estimate of rmsd is enough, if this algorithm can do such low precision calculation fast, that would be very helpful.